docs/try/installation.md

```diff
@@ -1,159 +1,47 @@
-Make sure that Git binary directory is in your computer path (environment variable).
-Under Windows, after installing Git you should have in your path environment variable:
-C:\Program Files\Git\cmd.
-Warning: TortoiseGit (https://tortoisegit.org/) does not install Git.
+### Download the source Code
 
-You can clone RINGMesh using:
+The source code is available by:
 
-```
-git clone https://github.com/ringmesh/RINGMesh/
-```
-
-## Linux
-
-
-### Configuring RINGMesh
-
-Execute cmake command in a RINGMesh/build directory.
-
-```bash
-mkdir build
-cd build
-```
-To configure using default options:
-```bash
-cmake ..
-```
-To define the options, use the cmake interface
-
-```bash
-cmake-gui .. or ccmake ..
-```
-
-### Compiling RINGMesh
-
-To compile RINGMesh, go to RINGMesh root directory and:
-
-```bash
-cd build/Release
-make
-```
-To build in debug, go to build/Debug instead.
-
-Note: RINGMesh uses C++ 11 features. You need gcc/g++ version higher or equal to 4.8 to compile it.
-
-### Generating the documentation
-
-If Doxygen is installed on your computer, a target ```doc-devkit``` is built during RINGMesh configuration. 
-You can generate the documentation using the following command in the `RINGMesh_root/build/config` directory:
-
-```bash
-make doc-devkit
-```
-Open the RINGMesh documentation with your favorite web browser at:  `RINGMesh_root/build/config/ringmesh/devkit/html/index.html`
-
-### Troubleshooting
-
-If you get this error during geogram gfx compilation (occured for Ubuntu 17.04):
-```
-No rule to make target '/usr/lib/x86_64-linux-gnu/libGL.so'
-```
-you can try under root:
-```bash
-rm /usr/lib/x86_64-linux-gnu/libGL.so
-ln -s /usr/lib/libGL.so.1 /usr/lib/x86_64-linux-gnu/libGL.so
-```
-### Additionnal information
-
-#### Eclipse-cdt project
-[Eclipse-cdt](http://www.eclipse.org/cdt/)
-project is provided (.project and .cproject). You can import RINGMesh into
-Eclipse: File>Import...>General>Existing Projects into Workspace.
-
-## Windows
-
-### Configuring RINGMesh
-
- * Launch CMake GUI interface. 
- * Indicate where is the source code as the path to RINGMesh root and where to put the binaries as 
- `RINGMesh_root/build`.
- * Set the Configuration options using the CMake GUI interface.
- * Launch the `configure`and `generate` option.
-
-RINGMesh compiles with the following Visual Studio version:
-
-* Visual Studio 12 2013 Win64
-* Visual Studio 14 2015 Win64
-* Visual Studio 15 2017 Win64
-
-Note: RINGMesh uses C++11 features. Make sure that you have installed C++ package for Visual Studio through the 
-Visual Studio installer.
-
-
-### Compiling RINGMesh
-
-RINGMesh need several third parties that are automatically compiled and installed by compiling 
-the project `SUPERBUILD.sln`.
-
- * Open the project `SUPERBUILD.sln` with the visual studio version that you choose during the configuration step.
- * Build the solution.
+ 1. cloning the directory from GitHub
+ 1. downloading Source archives (Released or lining) 
```

Comment:
what do you mean by "lining"?

Reply:
I means living but I am not sure about the terminology. we have released versions and a living one that aglomerate all new devs.